style.css

```diff
@@ -0,0 +1,111 @@
+body, html{
+    background-color: #4682b4;
+    margin: 0;
+    color: white;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+header{
```

Comment:
nitpick: make sure that the spacing around the curly brackets are consistent across the entire file